org-brain.el

```diff
@@ -935,7 +937,7 @@ Only get the body text, unless ALL-DATA is t."
         (org-remove-indentation entry-text)
       (with-temp-buffer
         (insert (org-remove-indentation entry-text))
-        (goto-char (org-brain-first-headline-position))
+        (goto-char (buffer-size))
```

Comment:
Now I haven't tested the code, but does `(buffer-size)` here work no matter the value of `org-brain-show-full-entry`?

Reply:
Fixed.